nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/blob/LocalBlobStoreConfiguration.java

```diff
@@ -45,6 +45,15 @@
 
     public static final String DEFAULT_PATH = "binaries";
 
+    /** In the initialization properties, the property for the store path strategy type (full|shortened). */
+    public static final String PROP_PATH_STRATEGY_TYPE = "pathStrategyType";
+
+    public static final String PATH_STRATEGY_TYPE_FULL = "full";
+
+    public static final String PATH_STRATEGY_TYPE_SHORTENED = "shortened";
+
+    public static final String DEFAULT_PATH_STRATEGY_TYPE = PATH_STRATEGY_TYPE_SHORTENED;
```

Comment:
I think we should keep PATH_STRATEGY_TYPE_FULL as default for 10.10.

Reply:
Yes this is necessary for compat. But I even suggest not doing that part of the change at all, see my comment in https://github.com/nuxeo/nuxeo-lts/pull/331#discussion_r773973855